example_package/README.md

```diff
@@ -0,0 +1,454 @@
+They are copied to the directory where the source code is compiled.
+All languages have the same additional files.
+
+```text
+extra_compilation_args:
+   cpp: ['abclib.cpp']
+
+extra_compilation_files: ['abclib.cpp', 'abclib.h']
+```
+
+## Time
+
+```text
+time_limit: 1000 # ms
+
+time_limits:
+  2: 2000
+  5: 7000
+```
+
+More precise time limit for each group or test can be defined in `time_limits` key.
+The more precise time limit has higher priority (first group, then global time limit).
+
+## Memory
+
+```text
+memory_limit: 262144 # kB
+
+memory_limits:
+  3: 131072
+  4: 131072
+```
+
+More precise memory limits can be defined in `memory_limits` key.
+Same as with time limits, the more precise memory limit has higher priority.
+
+## Title
+
+```text
+title: Przykładowy tytuł
+```
+
+Task title visible in the system.
+If there are Polish characters, they should be written for better readability.
+
+## Scores
+
+```text
+scores:
+  1: 20
+  2: 80
+```
+
+Number of points for each group can be defined in `scores` key.
+If this key is not specified, then all groups have the same number of points.
+(if number of groups doesn't divide 100, then the last groups will have the remaining points).
+Group 0 always has zero points.
+
+## Task ID
+
+```text
+sinol_task_id: abc
+```
+
+This key represents the short name (consisting of 3 letters) of the task.
+The names of files in `prog/`, `doc/`, `in/` and `out/` directories have to start with this task id.
+This key is only used by `st-make`: running `st-make export` creates
+an archive with the proper name, which sio2 uses as the task id.
+
+## Contest type
+
+```text
+sinol_contest_type: talent
+```
+
+sinol-make can behave differently depending on the value of `sinol_contest_type` key.
+Mainly, it affects how points are calculated.
+If the key is not specified, then (in st-make) `talent` is used. In sinol-make (OI version) is used 'default'.
+
+## Expected scores
+
+```text
+sinol_expected_scores: {}
+```
+
+st-make can check if the solutions run as expected when using `run` command.
+Key `sinol_expected_scores` defines expected scores for each solution on each tests.
+There should be no reason to change this key manually.
+It is automatically generated and managed by st-make.
+
+# in i out
+
+Są to foldery, w których znajdują się testy.
+Testy nazywamy `{TAG}{grupa}{nr_testu}.{in/out}`.
+
+Grupa:
+
+- 0, ocen - są to testy wstępne, nie liczą się do oceny i uczestnik ma do nich dostęp na zawodach.
+- 1,2,... - jest to grupa, punkty za nią dostaniemy jak przejdą wszystkie testy z danej grupy.
+
+nr_testu to kolejne litery alfabetu.
+A jak się skończą to dwie: a, ... z, aa, ab, ...
+
+Przykładowe nazwy to: `abc0a.in`, `abc1a.in`, `abc1b.out`, `abc3z.in`, `abc3aa.in`.
+
+Ciekawą formą nazywania jest też `{TAG}{grupa}t{nr}`, np `abc1t1.in`, jednak nie chce się przyjąć.
+
+**Testy ocen** - anomalią od tego są używane kiedyś testy ocen.
```

Comment:
kiedyś?

Reply:
Od teraz OI nie będzie dawać już testów ocen. Od 1 etapu ma być jak na OIJ czyli 0a, 0b, 0_. No chyba że tylko tak gadają, ale Tomek to forsuje więc imo przejdzie.